resources/views/page/donate.blade.php

```diff
@@ -0,0 +1,85 @@
+@extends('layout.default')
+
+@section('title')
+    <title>Donate - {{ config('other.title') }}</title>
+@endsection
+
+@section('meta')
+    <meta name="description" content="Donate">
+@endsection
+
+@section('breadcrumbs')
+    <li class="breadcrumbV2">
+        <a href="{{ route('pages.index') }}" class="breadcrumb__link">
+            Pages
+        </a>
+    </li>
+    <li class="breadcrumb--active">
+        Donate
+    </li>
+@endsection
+
+@section('main')
+    <h2>Donation Items</h2>
+    <div class="gird__container--donation">
+        @foreach ($items as $item)
+            <div class="grid__item--donation">
+                <div class="text-center oper-item card_header">
+                    <h1>{{ $item->name }}</h1>
+                </div>
+                <div class="text-center oper-item card">
+                    <span class="text-center">
+                        <p class="type">Type: {{ $item->type ?? 'N/A' }}</p>
+                        <p class="price">${{ $item->price_usd }}@if ($item->id <= 2)<sup>2</sup>@endif</p>
+                        <p class="adventage">{{ $item->seedbonus ?? 0 }} BON points</p>
+                        <p class="adventage">{{ $item->upload ?? 0 }} Upload</p>
+                        <p class="adventage">{{ $item->invites ?? 0 }} invites</p>
+                    </span>
+                </div>
+            </div>
+        @endforeach
+        <h2>Donate</h2>
+        <div class="text-center oper-item card_footer">
+            <form
+                class="form"
+                method="POST"
+                action="{{ route('create_payment') }}"
+                enctype="multipart/form-data"
+            >
+                @csrf
+                <label for="coin">Coin</label>
+                <select name="coin" class="form__select" x-model="coin">
+                    <option hidden disabled selected value=""></option>
+                    <option value="BTC">Bitcoin</option>
+                    <option value="XMR">Monero</option>
+                </select>
+                <hr>
```

Comment:
Please use the standard form select markup so that styles work properly

https://github.com/HDInnovations/UNIT3D-Community-Edition/blob/f03f660c062d5ec87e42be252febab65729783a5/resources/sass/components/form/select.scss#L1-L14

Reply:
Fixed in f9be0893930d5c245fdedfa850d2853764df7302